<launch>
    <param  name="stap_warp/base_frame" value="edo_base_link"/>
    <param  name="stap_warp/tool_frame" value="edo_link_ee_tip"/>
    <!-- <include file="$(find velocity_scaling_iso15066)/launch/publish_unscaled_joint_target.launch"/> -->
    <node name="blend_test" pkg="stap_warp" type="blend_test" respawn="false" output="screen" >
    </node> 
    <!-- launch-prefix="gdb -ex run  -args"> -->
    <!-- "gdb -.-ex run -.-args"  -->
</launch>